### ISSUE TITLE
Missing good descriptive text for the images in MailHog docs

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/freeCodeCamp#48723]
**Describe the bug**
In MailHog docs, we have three images that have alt text equal to `mailhog screenshot {{Number}}`, which isn't accessible. 

We can add descriptive alts, that describe what the images do for screen readers.

Here is the file https://github.com/freeCodeCamp/freeCodeCamp/blob/519e0d2a51221e54907b8ee0c21e09dce7457ee2/docs/how-to-catch-outgoing-emails-locally.md

**Comment by Sboonny** [comment]
To resolve this delete everything after, **except the useful links**

> Open a new browser tab or window and navigate to http://localhost:8025/ to open your MailHog inbox when the MailHog installation has completed and MailHog is running.

If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.
